pkg/database/database.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	resourceTableName        = "kubearchive.resource"
+	resourceTableName        = "resource"
```

Comment:
@ggallen is this change to use the public schema?

Reply:
Yes.

Reply:
I that case, I thought that we should have `public.resource`, or will it be public if we don't specify the schema used?

Reply:
I believe `public` is the default.